appcontainer/ImpersonationBarModel.js

```diff
@@ -46,6 +46,7 @@ export class ImpersonationBarModel {
         if (this.isOpen) {
             this.hide();
         } else {
```

Comment:
why wouldn't this go with the other throwIf in show()??

Reply:
It probably should.